spec/support/authenticated_session_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module AuthenticatedSessionHelper
+  def use_authenticated_current_user(options = {})
+    klass = options[:klass] || ApplicationController
```

Comment:
I think it's ok as long as you have a generic controller that is tested for auth already, and I think we have that. Otherwise just slows down specs.